src/ensembl/src/shared/components/loader/SidebarLoader.scss

```diff
@@ -0,0 +1,43 @@
+@import 'src/styles/common';
+
+.sidebarLoader {
+  box-sizing: border-box;
+  display: flex;
+  margin: 12px 0 0 10px;
+  flex-direction: column;
+  height: 100%;
+
+  section {
+    margin-top: 45px;
+  }
+
+  div {
+    background-color: $light-grey;
+    border-radius: 20px;
+    height: 9px;
+    margin-bottom: 10px;
+    animation: fading 0.5s ease-in;
+  }
+
+  .listHeading {
+    width: 50%;
+  }
+
+  .shortItem {
+    width: 70%;
+  }
+
+  .longItem {
+    width: 85%;
+  }
+}
+
+@keyframes fading {
```

Comment:
1) Probably `fade-in` rather than `fading?`
2) Could you check what the name of this animation in the final CSS file is? If it doesn't get modified by the CSS modules postprocessing, then it probably is a good idea to name it in a more specific way so that it doesn't collide with any other animation. Maybe `sidebar-loader-fade-in`?

Reply:
It changes `fade-in` to `fade-in__SidebarLoader__p01wg`

Reply:
Sounds good. Let's change fading to fade-in then.

Reply:
✅